FunctionalTableData/TableView/FunctionalTableData.swift

```diff
@@ -183,26 +169,26 @@ public class FunctionalTableData {
 	/// Returns the key path of the cell in a given `IndexPath` location.
 	///
 	/// __Note:__ This method performs an unsafe lookup, make sure that the `IndexPath` exists
-	/// before trying to transform it into a `KeyPath`.
+	/// before trying to transform it into a `ItemPath`.
 	/// - Parameter indexPath: A key path identifying where the key path is located.
 	/// - Returns: The key representation of the supplied `IndexPath`.
-	public func keyPathForIndexPath(indexPath: IndexPath) -> KeyPath {
+	public func keyPathForIndexPath(indexPath: IndexPath) -> ItemPath {
```

Comment:
`itemPathForIndexPath`?

Reply:
That would be a breaking change. I have more coming that would do stuff like that but want to minimize for now